index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Roland Portfolio</title>
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+        <link rel="stylesheet" href="css/style.css">
+    </head>
+
+    <body>
+        <div class="wrapper" id="wrapper">
+            <header class="header">
+                <nav class="nav">
+                    <img src="img/image_geometry_1.svg" alt="">
+                    <a href="#" onclick="showMenu()"><img class="menu-button" src="img/ic_menu.svg" alt="Menu Buttom"></a>
+                </nav>
+            </header>
+
+            <section class="headline">
+                <div class="title set">
+                    <h1>Roland<br> Ossisa Yuma</h1>
+                </div>
+                <div class="content set">
+                    <p class="description">
+                        Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+                        If you like what you see and have a project you need coded, don’t hesitate and contact me.
+                    </p>
+                    <button class="start-button">
+                        Start collaboration
+                    </button>
+                </div>
+                <div class="content-bottom">
+                    <img class="f-img" src="img/image_geometry_2.svg" alt="">
```

Comment:
- [ ] Good job on adding the button and styling it but kindly stick as much as you can to  [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9).

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/158685390-0eb42cbf-48bc-4215-9130-5f615c20b58a.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/158685423-8c7b9104-e2fb-4383-b600-fba6eb487311.png)